src/ScalableTypeInterface.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Dhii\Type;
+
+/**
+ * Represents a scalable type.
+ *
+ * @since [*next-version*]
+ */
+interface ScalableTypeInterface extends TypeInterface
```

Comment:
Same here. I think the proper name is "scalar" not "scalable".

Unfortunately due to the use of the word "scalable" in terms of web servers and PHP being a web-focused language, I can't find anything related to types.

But [on the manual they use "scalar"](http://php.net/manual/en/language.types.intro.php).